src/index.html

```diff
@@ -10,7 +10,57 @@
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>Search bar airbnb</h1>
+    <header class="header">
+
+      <form
```

Comment:
Don't add indents in such a way.
```suggestion
    <header class="header">
      <form
```